WiserTaskScheduler/WiserTaskScheduler/Modules/GenerateFiles/Services/GenerateFileService.cs

```diff
@@ -141,6 +149,40 @@ private async Task<JObject> GenerateFile(GenerateFileModel generateFile, List<in
                     Html = body
                 };
                 pdfFile = await htmlToPdfConverterService.ConvertHtmlStringToPdfAsync(pdfSettings);
+                
+                // Merge PDF's to generated PDF
+                if (generateFile.Body.MergePdfs.Where(p => !String.IsNullOrEmpty(p.WiserItemId)).ToArray().Length > 0)
+                {
+                    // Make stream of byte array
+                    var stream = new MemoryStream(pdfFile.FileContents);
```

Comment:
You should always put a `MemoryStream` in a `using` block (or dispose it in a try/finally), otherwise you might cause memory leaks and crashes.

Reply:
Added using to the declaration, according to https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/proposals/csharp-8.0/using